src/app/views/form/form.component.ts

```diff
@@ -59,13 +65,47 @@ export class FormComponent implements OnInit {
         this.evaluationId = this.evaluation.id;
       });
     }
+    this.charsRemaining$ = concat(of(''), this.comment.valueChanges).pipe(
```

Comment:
Why do you need `of('')` here?

Reply:
Since it concatinates two observables (Observable<string>, Observable<any>), I think this forces the outcome to be Observable<string>. This was also used in the lectures.